docs/html/development/getting-started.rst

```diff
@@ -26,7 +26,7 @@ Development Environment
 pip is a command line application written in Python. For developing pip,
 you should `install Python`_ on your computer.
 
-For developing pip, you need to install :pypi:`tox`. Often, you can run
+For developing pip, you need to install :pypi: `tox`_ . Often, you can run
```

Comment:
I’m quite sure this is not the right change. `:pypi:` is a custom derivative that renders this as a link to https://pypi.org/project/tox. Other similar occurrences apply.

Reply:
Oh! I see, I'm a newbie so thanks for helping. Can you suggest the correct syntax, please?

Reply:
The current one `:pypi:`tox` would be most appropriate (also remove the `_tox` ref at the bottom).

Reply:
Got it! Also, since you are so helpful, may I ask if I should close the PR in order to make the changes or there's a way

Reply:
You can simply change them back, commit, and push to update this pull request.